### PR TITLE
Enable footer on custom course certificates - EDLY-2595

### PR DESCRIPTION
**Pull request by taimoor-ahmed-1**
**Description:** This PR fixes the `logo_src` value for course certificates. 

**JIRA:** https://edlyio.atlassian.net/browse/EDLY-2595

**Related PR:** https://github.com/edly-io/edly-edx-themes/pull/262

### CODE REVIEW

File: lms/djangoapps/certificates/views/webview.py [hunk 1]
```diff
@@ -604,7 +604,6 @@ def render_html_view(request, user_id, course_id):
         _update_configuration_context(context, configuration)
 
         # Add certificate header/footer data to current context
-        context.update(get_certificate_header_context(is_secure=request.is_secure()))
         context.update(get_certificate_footer_context())
```
Comment (iam-mhaseeb) [inline lms/djangoapps/certificates/views/webview.py:607]: Why do we need to do this?

Comment (taimoor-ahmed-1) [reply lms/djangoapps/certificates/views/webview.py:606]: `get_certificate_header_context` is returning the static path for `logo_src` and `logo_url`, as we are already getting the correct values from `get_current_site_invalid_certificate_context` method, we need to remove this so values don't get overridden.

Comment (iam-mhaseeb) [reply lms/djangoapps/certificates/views/webview.py:606]: We should avoid changes in core files. Instead could you please check if we can somehow override with our function on priority?

Comment (taimoor-ahmed-1) [reply lms/djangoapps/certificates/views/webview.py:606]: We could either:
1. change name of values in our method `get_current_site_invalid_certificate_context()` and then use those everywhere in certificates.
2. or change the sequence of method calls `_update_context_with_basic_info` and `get_certificate_header_context` so correct values get overriden afterwards.

Comment (zubair-arbi) [reply lms/djangoapps/certificates/views/webview.py:606]: How about calling `get_current_site_invalid_certificate_context` after this piece of code?
```python
# Add certificate header/footer data to current context
context.update(get_certificate_header_context(is_secure=request.is_secure()))
context.update(get_certificate_footer_context())

context.update(get_current_site_invalid_certificate_context(default_html_certificate_configuration))
```

**Review by zubair-arbi — CHANGES_REQUESTED** [review]
I have added my feedback.

File: lms/djangoapps/certificates/views/webview.py (hunk 1 above)
Comment (taimoor-ahmed-1) [reply lms/djangoapps/certificates/views/webview.py:608]: Done.

**Review by zubair-arbi — APPROVED** [review]
LGTM 👍

**Review by iam-mhaseeb — APPROVED** [review]
👍

**Pull request merged**